common/djangoapps/cme_registration/tests/registration_tests.py

```diff
@@ -223,6 +223,16 @@ def test_sub_specialty_not_required(self):
 
     @unittest.skipIf(settings.FEATURES.get('DISABLE_CME_REGISTRATION_TESTS', False),
                      dedent("""Skipping Test because the url is not in CMS"""))
+    def test_affiliation_required(self):
+
+        self.post_vars['affiliation'] = ''
+        url = reverse('create_account')
+        response = self.client.post(url, self.post_vars)
+
+        self.assertContains(response, '{"field": "affiliation", "value": "Choose your affiliation", "success": false}')
```

Comment:
This is a bit brittle. If any of that response output changes, from message text to even adding a new, unrelated key to the response, all of these tests would need updated.

If we just parse the response from JSON to a dict/list, we could explicitly check for specific key/values like:
```python
# Assume response is a list of dicts
import json
response = json.loads(response)
does_require_affiliation = any(
    True
    for error in response
    if error['field'] == 'affiliation' and not error['success']
)
self.assertTrue(does_require_affiliation)
```

Not a blocker, since this convention is used elsewhere.